bucket/sysinternals.json

```diff
@@ -7,7 +7,7 @@
         "url": "https://docs.microsoft.com/en-us/sysinternals/license-terms"
     },
     "url": "https://download.sysinternals.com/files/SysinternalsSuite.zip",
-    "hash": "c220d270006ef3e9e7ad521be7880b438b0d5285655fb4e40eebc1d246851165",
+    "hash": "7414727a4714caaac5d2194d44a94fa130aa893cfccee95d4a65a7613ea9cdab",
```

Comment:
I have a different hash than this one:

```
Checking hash of SysinternalsSuite.zip ... ERROR Hash check failed!
App:         extras/sysinternals
URL:         https://download.sysinternals.com/files/SysinternalsSuite.zip
First bytes: 50 4B 03 04 14 00 00 00
Expected:    c220d270006ef3e9e7ad521be7880b438b0d5285655fb4e40eebc1d246851165
Actual:      1db197a9b449ad99bbf98147b7253478f3f1cbf33b65c96d5c76c0e7198620c1
```

Reply:
```
$ sha256sum.exe ~/Downloads/SysinternalsSuite.zip
1db197a9b449ad99bbf98147b7253478f3f1cbf33b65c96d5c76c0e7198620c1 *Downloads/SysinternalsSuite.zip
```

Reply:
It will change multiple times, until all hotfixes are shipped.